press/press/doctype/team/team.py

```diff
@@ -124,19 +124,24 @@ def create_new(
 
 		team.insert(ignore_permissions=True, ignore_links=True)
 		team.append("team_members", {"user": user.name})
-		team.append("communication_emails", {"type": "invoices", "value": user.name})
-		team.append(
-			"communication_emails", {"type": "marketplace_notifications", "value": user.name}
-		)
+		if not account_request.invited_by_parent_team:
+			team.append("communication_emails", {"type": "invoices", "value": user.name})
+			team.append(
+				"communication_emails", {"type": "marketplace_notifications", "value": user.name}
+			)
+		else:
+			team.parent_team = account_request.invited_by
 		team.save(ignore_permissions=True)
 
-		team.create_stripe_customer()
+		if not account_request.invited_by_parent_team:
```

Comment:
Invited teams should also be allowed to create their own sites. This isn't required.

Reply:
@rutwikhdev They can create their own site but since the billing is handled via parent team thought this step isn't required. Thoughts?

Reply:
Let's still keep it? It's just a stripe object for unique customer. In case the child team wants to branch out from parent in future, you shouldn't need to create this stuff manually again.